backend/czi_hosted/data_common/matrix_loader.py

```diff
@@ -244,6 +48,7 @@ def __init__(self, location, matrix_data_type=None, app_config=None):
 
             self.matrix_type = CxgAdaptor
 
+    # TODO @mdunitz remove when removing conversion code, also remove server_config.multi_dataset__allowed_matrix_types
```

Comment:
For any todos in code, make sure you create a new issue in the cellxgene repo and tag the issue number here so that we don't forget to eventually address the todos.